src/modules/billing/controllers/invoices.js

```diff
@@ -14,4 +14,15 @@ const patchInvoice = async request => {
   }
 };
 
+const resetIsFlaggedForRebilling = async request => {
+  try {
+    const { batchId } = request.params;
+
+    return await invoiceService.resetIsFlaggedForRebilling(batchId);
```

Comment:
You will need to add another method to only cancel the reissue of a specific invoice. This will use the invoice id received form the route